geoutils/georaster.py

```diff
@@ -617,6 +617,13 @@ def copy(self: RasterType, new_array: np.ndarray | None = None) -> RasterType:
 
         return cp
 
+    @property
+    def __array_interface__(self) -> dict[str, Any]:
+        if self._data is None:
+            self.load()
```

Comment:
Should we really load it? Maybe it's not wanted by the user (some cropping or loading only a single band might be needed instead). I would rather raise an error.